src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/WorkbookRepository.kt

```diff
@@ -107,7 +107,8 @@ class WorkbookRepository(private val db: IDatabaseAccessors) : IWorkbookReposito
                 throw IllegalStateException("Content text is null for resource")
             }
             TextItem(content.text ?: "[empty]", MimeType.of(format))
-        } ?: throw IllegalStateException("Content format is null")
+        } ?: TextItem(content.text ?: "[empty]", MimeType.of("usfm")) // TODO 7/5: temporary workaround
```

Comment:
What's wrong with using format now?

Reply:
The problem is that parseUsfm is broken, so 'format' is not being written to the content entries. (There is a TODO comment a few lines above this.) I didn't want the app to crash until we fix parseUsfm